### ISSUE TITLE
Question: Is this project still active?

### ISSUE THREAD

**Issue by michaeltlombardi** [madewokherd/oneget-python#1]
If so, is there any intent to package a release?

I'd like to be helpful but I've got zero C# mastery. What can I do to help?

**Comment by madewokherd** [comment]
Hi,

Unfortunately, there's no way to package it right now, at least if we want it to be possible to bootstrap the provider through oneget. They don't currently have a packaging solution that allows shipping a C# dll with other files. We really need the dll and some python source files.

There's no reason I couldn't continue working on it without the ability to bootstrap, but it makes it hard to find the motivation. So I've been waiting for the oneget project to solve this.

**Comment by michaeltlombardi** [comment]
That makes sense, but is frustrating. Is there an issue with them for this roadblock? I imagine it's also going to kill attempts at ruby/node/etc providers.

I can't in good conscience want you to keep working on something that might not be releasable via normal channels. 

What state is it in now? If I was hellbent on using it, could I? I'd like to document this process if so.

**Comment by madewokherd** [comment]
I don't know if there's an issue with them. I know it was discussed in meetings with them.

Having thought about it again, maybe it'd be possible to embed the resources we need in the .dll and extract them on first run. I'll have to research that.

Either way, we should probably file an issue with them for getting the provider installable and in the bootstrap database, so I can always know what the next step is.

Yes, it should be possible to build and install the project manually. When you build the dll in visual studio, the other required files will be in a directory with it. You will then have to register the provider manually (see https://github.com/OneGet/oneget/wiki/Provider-assembly-search-locations).

Note that I haven't tested it recently, and PyPI, cpython, or the python website may have changed in a way that breaks it.

**Comment by michaeltlombardi** [comment]
I'm going to look into seeing if it's possible do the build/registration etc on a remote box (in this case, my build server) programmatically and I'll let you know how it goes. May be a delay of up to three weeks - for now I'm going to do the quick and dirty of just running a script to install the packages I need as a post-deployment piece.

I'd like to roll it all up into a DSC config on the build server instead, but that's longer term.

TLDR; I'm going to give this a shot and let you know what I find.
